src/app.jsx

```diff
@@ -508,8 +519,30 @@ const NavigatorCardBody = ({
                     return [sortedFiles[newIdx]];
                 });
             } else if (e.key === "Enter" && selected.length === 1) {
+                console.log("Entered");
                 onDoubleClickNavigate(selected[0]);
             }
+            else if (e.key === "Delete") {
+                const currentDir = path.join("/") + "/";
+                if(selected.length===1){    
+                const selectedContext=selected[0];
```

Comment:
Global: Please fix the formatting, linting errors should show up in the terminal where you run `make check`. 
I suggest you set up 'fix linting on save' in your editor to fix this automatically

Reply:
@MahmoudHamdy02 Thanks for the feedback I'll fix the formatting and about deleting multiple items I know it's not the most efficient and I know that it already supports it but I tried few things and wierd erros were popping after deleting multiple items